integration-tests/docker/docker-compose.azure.yml

```diff
@@ -0,0 +1,182 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
What is the purpose of this since we already have integration-tests/docker/docker-compose.override-env.yml

Note that the env file in environment-configs/override-examples/ is never meant to be use as-is or modified directly. User should copy this and write their own override env file and use integration-tests/docker/docker-compose.override-env.yml

Reply:
Usually developers prepare docker-compose files for each "use case". And all this docker-compose files it's something stable, you do some changes in this files rarely.
If you have all docker-compose files for each use case you can see how many combinations you have, you can see what the difference between each type of configuration.

I understand that all files in "environment-configs/override-examples/" this is more like example configuration but for new developer who want to understand how to run "s3" tests he need to check only docker-compose.s3.yml.

We have 2 ways how to run "druid containers for s3" tests:
1. run hadoop + druid: 
docker-compose -f ./integration-tests/docker/docker-compose.s3.yml up -d
2.run hadoop: 
docker-compose -f ./integration-tests/docker/docker-compose.hadoop.yml up -d
run druid-containers:
OVERRIDE_ENV=<full_path_to_env_file> docker-compose -f ./integration-tests/docker/docker-compose.override-env.yml up -d

Reply:
I am fine to remove all "azure", "gcs", "s3" docker-compose files but to run docker containers by docker-compose we need to run 2 commands or create own "docker-compose" files for each run configuration.

Reply:
I think "azure", "gcs", "s3" docker-compose are not necessary and can add to confusion. Those "environment-configs/override-examples/" are examples. Even if you do want to run the s3 tests, you will likely need to modify the files from "environment-configs/override-examples/". Also, I think it make more sense to separate hadoop from Druid clusters. Our configurations for Druid can simply be running with default env and running with override env. Then hadoop can optionally be started or not started.

Reply:
removed docker-compose.s3.yml, docker-compose.azure.yml, docker-compose.gcs.yml files